java/common/pom.xml

```diff
@@ -85,6 +85,7 @@
     <dependency>
       <groupId>org.wso2.carbon</groupId>
       <artifactId>org.wso2.carbon.utils</artifactId>
+      <scope>provided</scope>
```

Comment:
Why do we need to make this as provided?

Reply:
If not carbon utils will be bundled inside the shindig.war